### ISSUE TITLE
Get airflow certbot auto-renewing as expected 

### ISSUE THREAD

**Issue by frankhereford** [cityofaustin/atd-data-tech#18779]
I received an email from CertBot today stating that we’re going to have two certs expire in 19 days, which I did not expect.

> Your certificate (or certificates) for the names listed below will expire in 19 days (on 2024-09-14). Please make sure to renew your certificate before then, or visitors to your web site will encounter errors.
>[airflow-workers.austinmobility.io](http://airflow-workers.austinmobility.io/)
>[airflow.austinmobility.io](http://airflow.austinmobility.io/)

This issue is to track the discovery required to figure out why these are not auto-renewing and to fix it.

**Comment by frankhereford** [comment]
## TLDR

Turns out that this was related to SELinux, which is the enhanced security system used by RHEL. Leaning on my robot buddy to summarize this work:

## SELinux

SELinux (Security-Enhanced Linux) is a security architecture integrated into the Linux kernel that provides mandatory access control. It defines the access and transition rights of every user, application, process, and file on the system. SELinux operates on the principle of least privilege, allowing processes to access only the resources they need to function.

## Work Completed

We identified that the cron job file was a symlink pointing to a location with an incorrect SELinux context (var_t instead of system_cron_spool_t). We used semanage and restorecon commands to set and apply the correct SELinux context (system_cron_spool_t) to the target file. Finally, we corrected the formatting of the cron job entries to ensure they comply with the expected syntax for /etc/cron.d/ files.

## Next actions

The cronjob should kick off here at noon local time, so I have not manually renewed the cert, and I'm going to put this in "review" until the afternoon, at which point, I'll confirm that it worked and close it or continue remediation. @mddilley, thanks for the sync on this today, and please LMK if you'd like to change the direction here at all. 🙏

**Comment by frankhereford** [comment]
Additionally, the certbot "installation" on `on-prem-1` has had the email address associated with domains renewed changed.

```
Your e-mail address was updated to transportation.data@austintexas.gov.
```

**Comment by frankhereford** [comment]
<img width="505" alt="image" src="https://github.com/user-attachments/assets/159c8d20-4333-468c-b672-441aa20974d8">

This was fixed by correctly identifying the crontab to SELinux. The cert renewed itself via cron today at noon as expected.
